client/boot/index.js

```diff
@@ -154,15 +154,14 @@ function boot() {
 		i18n.setLocaleSlug( user.get().localeSlug );
 	} );
 
-	// Temporary support for development of the Support User feature
-	if ( config.isEnabled( 'support-user' ) ) {
-		require( 'lib/user/dev-support-user' )( user );
-	}
-
 	translatorJumpstart.init();
 
 	reduxStore = createReduxStore();
 
+	if ( config.isEnabled( 'support-user' ) ) {
```

Comment:
Should we have some other check here to not include this file for regular usage?